### ISSUE TITLE
Add support for ngRequired

### ISSUE THREAD

**Issue by alainlecluse** [Toilal/ng-pickadate#12]
It would be nice to have support for ngRequired.

**Comment by Toilal** [comment]
ng-required should already work if

1) `<input>` tag is embedded in a `<form>` tag
2) text value is binded to the controller using `ng-model`

This is not really related to this library, this is just how validation works in AngularJS, as this directive is defined on a standard `<input type="text" ...>` tag.

Could you confirm it works ?

**Comment by alainlecluse** [comment]
Hi Toilal,

that would seem likely. 
due to your answer in #10 

> ng-model will bind the text value like any other text input, so it's not recommanded to use it along with ng-pickadate.
> I removed ng-model.

I'm going to make a example somewhere to illustrate it.

**Comment by Toilal** [comment]
Add `ng-model` back, but don't use it. It will be used by AngularJS internally only for validation, but trying to modify this value from controller can break the normal behavior of `pickadate` directives.

I didn't test, but this should work :

``` html
<form name="validationForm">
    <input type="text" name="dateField" ng-model="curDateText" pickadate="curDate" required >
    <div ng-show="validationForm.dateField.$error.required">Date is required.</div>
</form>
```

EDIT: `required` is enough, you may use `ng-required` if the requirement vary. But I don't think it's a common use case.

**Comment by Toilal** [comment]
It's working properly in the latest released version (v0.2.0), and with example on the website : http://toilal.github.io/ng-pickadate/
